aten/src/ATen/native/Blas.cpp

```diff
@@ -137,6 +140,110 @@ Tensor dot(const Tensor &self, const Tensor &other){
   });
 }
 
+// this is to port LDA_COND from the TH codebase
+// #define LDA_COND(M, N, LDA) ((N) == 1 || (LDA) >= THMax(1, (M)))
+// lda_cond function in this file is not used
+// because it requires `lda > std::max<int64_t>(1L, m)`
+// instead of `>=` as defined in the TH implementation
+constexpr inline bool lda_cond_addr(int64_t m, int64_t n, int64_t lda) {
```

Comment:
In addition to the note above let's add a brief comment describing this function

Reply:
sure, I will add a brief comment.